### PR TITLE
docs: Added basic description of validation in nti.

### PR DESCRIPTION
**Pull request by stian-sandvold**

### CODE REVIEW

File: src/commonmark/en/content/developer/web-api/new-tracker.md [hunk 1]
```diff
@@ -1039,25 +1039,55 @@ There are various error codes for different error scenarios. The following table
...
+- Geometry (The coordinates must match the format as specified by it's type)
+
+#### User access
+All data imported will be validated based on the metadata of the data imported (Sharing) and the organisation unit of the data imported (Organisation Unit Scopes). More information about sharing and organisation unit scopes can be found in the following sections.
+
+Sharing is validated at the same time as references are looked up. That means a references the user cannot access, will be treated as if it doesn't exist. Sharing is checked for each metadata referenced in the data being imported.
+
+Organisation units on the other hand, serve a dual purpose. Primarily, it will make sure that data can only be imported when it's imported for an organisation unit the user have within their "capture scope". Secondly, organisation units are also used to restrict what programs are available. That means if you are trying to import data for an organisation unit which does not have access to the program you are importing, the import will be invalid.
+
+User's with the `ALL` authority transcends, in these cases, validations related to the user, but not programs.
```
Comment (vilkg) [inline src/commonmark/en/content/developer/web-api/new-tracker.md]: I don't fully understand what you mean here. Maybe add a sentence or two explaining it? User's - users?

File: src/commonmark/en/content/developer/web-api/new-tracker.md [hunk 2]
```diff
@@ -1039,25 +1039,55 @@ There are various error codes for different error scenarios. The following table
...
+
+#### Attribute and Data values
+
+Attributes and data values are part of a tracked entity and an event respectively. However, attributes can be linked to a tracked entity either through it's type (TrackedEntityType) or it's program (Program). Additionally, attributes can also be unique.
```
Comment (vilkg) [inline src/commonmark/en/content/developer/web-api/new-tracker.md]: it's - its?

File: src/commonmark/en/content/developer/web-api/new-tracker.md [hunk 3]
```diff
@@ -1039,25 +1039,55 @@ There are various error codes for different error scenarios. The following table
...
+
+The initial validation done in the import, is to make sure the value provided for an attribute or data element conforms to the type of value expected. For example, if you import a value for a data element with a numeric type, the value is expected to be numeric. Any errors related to a mismatch between a type and a value will result in the same error code, but with a specific message related to the type of violation.
```
Comment (vilkg) [inline src/commonmark/en/content/developer/web-api/new-tracker.md]: Would be good to mention that validation also depends on the validationStrategy set on program? ON_COMPLETE, ON_UPDATE_AND_INSERT and etc.

File: src/commonmark/en/content/developer/web-api/new-tracker.md (hunk 1 above)
Comment (stian-sandvold) [reply src/commonmark/en/content/developer/web-api/new-tracker.md]: Completely agree. I was a bit unsure how to easily explain it, but I see now it's a bit too compact. Will make it a bit more clear :)

File: src/commonmark/en/content/developer/web-api/new-tracker.md (hunk 2 above)
Comment (stian-sandvold) [reply src/commonmark/en/content/developer/web-api/new-tracker.md]: Will run through language check now :)

File: src/commonmark/en/content/developer/web-api/new-tracker.md (hunk 3 above)
Comment (stian-sandvold) [reply src/commonmark/en/content/developer/web-api/new-tracker.md]: Great feedback! This is the kind of details I might overlook. Will add info

**Pull request merged**